helpers/calendar/ratios_and_utilization.py

```diff
@@ -72,13 +75,20 @@ def get_analytics_ratios_per_room(self, query, start, end):
                                                         app_events_list)
 
             ratio_object = RoomAnalyticsRatios().map_results_to_ratio_class(
-                                checkins,
-                                cancellations,
-                                calendar_events_list,
-                                app_events_list,
-                                room['name'])
+                checkins=checkins,
+                cancellations=cancellations,
+                calendar_events_list=calendar_events_list,
+                app_events_list=app_events_list,
+                room_name=room['name'],
+                room_id=room['room_id'])
 
             response.append(ratio_object)
+
+        if room_id:
+            for room_response in response:
+                if room_response.room_id == room_id:
+                    response = room_response
+
         return response
```

Comment:
@jesseinit 
Consider adding a `break` statement after `response = room_response`

Reply:
Thanks @rajeman

Reply:
> @jesseinit
> Consider adding a `break` statement after `response = room_response`

@rajeman I'm not sure what you're suggesting here. Why is the `break` statement necessary?

Reply:
So that the `for loop` will be terminated immediately he gets the desired `room_response`

Reply:
I see it. Nice suggestion @rajeman